terraform/modules/paas/main.tf

```diff
@@ -60,6 +60,13 @@ resource cloudfoundry_route web_app_cloudapps_digital_route {
   hostname = local.web_app_name
 }
 
+resource cloudfoundry_route web_app_cloudapps_digital_route2 {
```

Comment:
As discussed IRL, we could build a list of cloudapps hostnames and use it in the route resource above

Reply:
done